bears/general/FilenameBear.py

```diff
@@ -12,6 +12,7 @@ class FilenameBear(LocalBear):
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
```

Comment:
nah, just leave it empty for this one, this isn't formatting. Also it'll check the formatting box for all programming languages which is just wrong.